src/headless/main.cpp

```diff
@@ -72,9 +112,24 @@ static bool run_headless(Dataloader::path_vector_t const& roots, bool run_tests)
 		CountryInstanceManager const& country_instance_manager =
 			game_manager.get_instance_manager()->get_country_instance_manager();
 
-		print_ranking_list("Great Powers", country_instance_manager.get_great_powers());
+		std::vector<CountryInstance*> const& great_powers = country_instance_manager.get_great_powers();
+		print_ranking_list("Great Powers", great_powers);
 		print_ranking_list("Secondary Powers", country_instance_manager.get_secondary_powers());
 		print_ranking_list("All countries", country_instance_manager.get_total_ranking());
+
+		Logger::info("===== RGO test... =====");
+		for (int i = 0; i < std::min<int>(3, great_powers.size()); ++i) {
+			CountryInstance const& great_power = *(great_powers[i]);
```

Comment:
```suggestion
			CountryInstance const& great_power = *great_powers[i];
```
Brackets not required as `var[...]` has higher precedence than `*var`